src/pages/signup-affiliates/components/wizard-component/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import WizardFooter from './wizard-footer'
+import WizardHeader from './wizard-header'
+import Stepper from './stepper'
+import { WizardComponentsProps } from 'pages/signup-affiliates/_types'
+import { useDebouncedEffect } from 'components/hooks/use-debounced-effect'
+import { useIsRtl } from 'components/hooks/use-isrtl'
+import device from 'themes/device'
+
+export const Background = styled.div`
+    position: absolute;
+    width: 100%;
+    block-size: 100vh;
+    top: 0;
+    left: 0;
+    background-color: var(--color-black);
+    opacity: 0.4;
+    z-index: 5;
+
+    @media ${device.tabletL} {
+        block-size: 100%;
+        opacity: 0;
+        z-index: unset;
+    }
+`
+export const Modal = styled.div<{ is_rtl?: boolean }>`
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    justify-content: space-between;
+    top: 50%;
+    left: ${({ is_rtl }) => (is_rtl ? '-20%' : '50%')};
+    block-size: 80%;
+    min-inline-size: 70%;
+    border-radius: 8px;
+    background-color: white;
+    transform: translate(-50%, -50%);
+    z-index: 6;
+    overflow: hidden;
+
+    @media ${device.tabletL} {
+        position: unset;
+        transform: unset;
+        border-radius: unset;
+        min-inline-size: unset;
+        inline-size: 100vw;
+        block-size: 100svh;
+        overflow: scroll;
+        padding-block-end: 150px;
+    }
+`
+const Container = styled.div`
+    overflow: scroll;
+`
+const Content = styled.div`
+    block-size: fit-content;
+    overflow-x: auto;
+`
+const WizardComponent = ({
+    children,
+    show_wizard,
+    setSignupStatus,
+    onSubmit,
+    step,
+    setStep,
+    next_btn_enabled,
+    setNextBtnEnabled,
+}: WizardComponentsProps) => {
+    useEffect(() => {
+        setNextBtnEnabled(next_btn_enabled)
+    }, [next_btn_enabled])
+    const is_rtl = useIsRtl()
+
+    useDebouncedEffect(
+        () => (document.body.style.overflow = show_wizard ? 'hidden' : 'unset'),
+        [show_wizard],
+        1,
```

Comment:
Why debounce? Why 1?

Reply:
got from app.deriv.com by another developer. I'll pay attention on it 👍

Reply:
Actually deprecated and unused code :D I'll remove it